common/app/components/Footer/index.js

```diff
@@ -1 +1 @@
-export default from './Footer.jsx';
```

Comment:
This is valid syntax when using https://github.com/leebyron/ecmascript-export-default-from